src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,54 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import React from 'react';
+
+export type Tab = {
+  id: string,
+  title: string,
+  content: string,
+};
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tabId: string) => void,
+  getSelectedTab: () => Tab | undefined,
```

Comment:
you don't need it
pass selected tab or perform a by-id search in this component